### ISSUE TITLE
Hotspots

### ISSUE THREAD

**Issue by mpvlc** [rajeevgade/vue-360#15]
Is it possible to add tracking hotspots ? 
Something like this
[https://codepen.io/pixelthing/pen/EgAJWr](https://codepen.io/pixelthing/pen/EgAJWr)

**Comment by rajeevgade** [comment]
@mpvlc Yes. This has been something that was on my mind before and I started working on this long ago but just couldn't finish it coz of time issues. Hopefully, I can find time sooner and work on it. It is already added to the roadmap of this package. If anyone is willing to contribute, I'd be happy to accept their pull request changes coz I don't know how long it will take for me to find a good time to work on this feature.
